docs/explanation/security.md

```diff
@@ -0,0 +1,25 @@
+# Security
+
+A software layer between your application and database, such as SQLair, can be a
+potential security threat. Issues with SQLair could lead to exploits such as SQL
+injection or Denial-of-Service through panics or high CPU usage. SQLair
+takes a variety of measures to protect users against these attack vectors.
+
+## Argument sanitization
+The contents of query arguments extracted by SQLair are passed as parameters to
+the database rather than injected into the query as strings. 
+
```

Comment:
So what does this mean for security? Anything the user should do?

Reply:
Added a sentence about how this mitigates against SQL injection.